syntax = "proto3";

option go_package = "gen/go/media_downloader";

service MediaDownloader {
  rpc RequestMedia(RequestMediaRequest) returns (RequestMediaResponse) {}
  rpc ReleaseMedia(ReleaseMediaRequest) returns (ReleaseMediaResponse) {}
}

message RequestMediaRequest {
  string media_uri = 1;
}

message RequestMediaResponse {
  string media_path = 1;
}

message ReleaseMediaRequest {
  string media_uri = 1;
}

message ReleaseMediaResponse {
}